detox/local-cli/test.js

```diff
@@ -115,9 +115,25 @@ module.exports.builder = {
     alias: 'workers',
     group: 'Execution:',
     describe:
-      '[iOS Only] Specifies number of workers the test runner should spawn, requires a test runner with parallel execution support (Detox CLI currently supports Jest)',
-    default: 1,
-    number: true
+      '[iOS Only] Specifies the number of workers the test runner should spawn, requires a test runner with parallel execution support (Detox CLI currently supports Jest)',
+    coerce(param) {
+      let workers = 1;
```

Comment:
@sidferreira, why not just remove `number: true` and convert to string `default: "1"`?

Reply:
No specific reason. TBH I was wondering if should add the `os.cpus()` here to increase the validation.

Reply:
Why the validation should be increased? Are there edge cases?

Reply:
Thinking like this, your change is indeed enough